views.py

```diff
@@ -741,6 +741,9 @@ def typesetting_typesetter_download_file(request, assignment_id, file_id):
         or file.pk in assignment.round.article.supplementary_files.values_list(
             'file__pk', flat=True,
             )
+        or file.pk in assignment.round.article.galley_set.values_list(
+                'file__pk', flat=True,
+            )
```

Comment:
This will grant the typesetter access over galley files even if they haven't been added by them, while before they would only have access to files owned by them. Is that alright? It is read access anyway so I suppose it is just fine